docs/mkcloud.md

```diff
@@ -20,6 +20,14 @@ or testing purposes.
   $ sudo service libvirtd start  # to start the daemon
   ```
 
+  If you are using `libvirtd >= 1.3.0` then you should also start the
+  `virtlogd` service.
+
+  ```
+  $ sudo service virtlogd status # to check the status
+  $ sudo service virtlogd start  # to start the daemon
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

On 2016-07-21 15:13, Markos Chandras wrote:
> Also=virtlogd.socket |
> 
> so my understanding is that if you |enable| the libvirt service
> file then virtlogd will also be enabled and both will be started on
> next reboot right?

maybe it would even be started without reboot, just with a
rclibvirtd restart

The other open question is if you can use `service` to start a .socket
- - did you test it?
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2

iEYEARECAAYFAleQzm4ACgkQSTYLOx37oWQ7BgCdEMwFeErgFymltKxf0F+57VTZ
XXcAn0DkqLRJUAqgJ6z7Rq0d2HDqqd4B
=y507
-----END PGP SIGNATURE-----